org.civicrm.bootstrapcivihr/gulpfile.js

```diff
@@ -22,7 +22,7 @@ gulp.task('sass', ['sass:sync'], function () {
     .pipe(postcss([postcssPrefix({
       prefix: bootstrapNamespace + ' ',
       exclude: [
-        /^html/, /^body/, /page-civi/, /crm-container/, outsideNamespaceRegExp
+        /^html/, /^body/, /page-civi/, /crm-container/, /spinner/, outsideNamespaceRegExp
```

Comment:
Is that what we usually do? Last time there was a discussion that this should not be like that )or maybe I am missing some point). cc @AkA84

Reply:
@igorpavlov 
Actually, previously something like following was used:
```
[ /^html/, /^body/, /page-civi/, ".crm-container .other-class", ".crm-container .one-other-class"]
```
And after our discussion we decided to use following instead considering it as better approach:
```
[/^html/, /^body/, /page-civi/, /crm-container/]
```
So, this is the approach we will be following.

cc @AkA84

Reply:
why is this even necessary?

Reply:
@AkA84 It's necessary as the spinner is used outside the `bootstrap-theme` namespace scope in hrjobcontract and if the `spinner` styles are namespaced within the `bootstrap-theme` in bootstrapcivihr the styles will not be available in hrjobcontract.

Reply:
@bmnepali please check what `outsideNamespaceRegExp` does

Reply:
@AkA84 thank!